public/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8">
+    <title>Color Match Game</title>
+        <link href="/css/styles.css" rel="stylesheet" />
+  </head>
+<body>
+<h1>Color Drop</h1>
+<br>
+<div id="container">
+  <form name="playerform"><label for="key">Player Name: </label><br />
+  <input id="key" name="key" type="text"/><br />
+  <input id="save" type="button" value="Pick Colors!"/>
+  <ul id='list'></ul>
+</div>
```

Comment:
This closes the #container div. I'm only guessing this isn't what you intended, because you have a view in control of this container which has the click handler for the reset button. However, since the reset button isn't in this div, the click handler doesn't handle the click event on the reset button. This is why I couldn't figure out where the click handler was, because the click handler that is ostensibly set up to handle it doesn't do anything when that click actually happens.